addrmgr/addrmanager.go

```diff
@@ -748,20 +746,8 @@ func (a *AddrManager) reset() {
 //
 // This function is safe for concurrent access.
 func (a *AddrManager) HostToNetAddress(host string, port uint16, services wire.ServiceFlag) (*NetAddress, error) {
-	// Tor address is 16 char base32 + ".onion"
 	var ip net.IP
-	if len(host) == 22 && host[16:] == ".onion" {
-		// go base32 encoding uses capitals (as does the rfc
-		// but Tor and bitcoind tend to user lowercase, so we switch
-		// case here.
-		data, err := base32.StdEncoding.DecodeString(
-			strings.ToUpper(host[:16]))
-		if err != nil {
-			return nil, err
-		}
-		prefix := []byte{0xfd, 0x87, 0xd8, 0x7e, 0xeb, 0x43}
-		ip = net.IP(append(prefix, data...))
-	} else if ip = net.ParseIP(host); ip == nil {
+	if ip = net.ParseIP(host); ip == nil {
```

Comment:
I removed it in the final commit. Ultimately, do you want me to squash this entire PR down into one commit, or keep it as 3? That'll inform how I proceed with resolving your feedback--should I just keep adding commits (expecting to squash them all at the end), or go back and edit commits and force-pushing (which makes it harder for reviewers)?

Reply:
Keeping separate commits for each logical change is preferred.

I don't mind whichever approach you want to take with adding commits to address review items now that are squashed later or force pushing.  I find them equally easy to review because github gives you a "force-pushed" link with the diff whenever commits are updated anyway.

I personally keep the commits updated as I go rather than having to figure out how to squash them into the right commits later, but whatever works best for you is fine.

Reply:
Updated.